modules/Db2/scripts/install_Db2.sh

```diff
@@ -90,75 +98,149 @@ if ${OC_CMD} get catalogsource -n openshift-marketplace | grep ibm-operator-cata
     done
 echo
 
+## 
+## Creating DB2 Operator Group 
+##
+echo
+echo "Creating Operator Group object for DB2 Operator"
+cp  ../../modules/Db2/templates/db2-operatorgroup.template.yaml  ../../modules/Db2/files/db2-operatorgroup.yaml
+sed -i.bak "s|paramDB2Namespace|$DB2_PROJECT_NAME|g" ../../modules/Db2/files/db2-operatorgroup.yaml
+oc apply -f ../../modules/Db2/files/db2-operatorgroup.yaml
+
+
 ###### Create subscription to Db2 Operator
 echo -e "\x1B[1mCreating the Subscription...\n${DB2_SUBSCRIPTION_FILE}\n\x1B[0m"
 kubectl apply -f ${DB2_SUBSCRIPTION_FILE}
 
-echo "Sleeping for 5 minutes"
-sleep 150
+echo
+echo "Creating Subscription object for DB2 Operator"
+cp ../../modules/Db2/templates/db2-subscription.template.yaml ../../modules/Db2/files/db2-subscription.yaml
+
+sed -i.bak "s|paramDB2Namespace|$DB2_PROJECT_NAME|g" ../../modules/Db2/files/db2-subscription.yaml
+sed -i.bak "s|paramDB2OperatorVersion|$DB2_OPERATOR_VERSION|g" ../../modules/Db2/files/db2-subscription.yaml
+sed -i.bak "s|paramDB2OperatorChannel|$DB2_OPERATOR_CHANNEL|g" ../../modules/Db2/files/db2-subscription.yaml
+oc apply -f ../../modules/Db2/files/db2-subscription.yaml
 
-###### Create subscription to Db2 Operator
-echo -e "\x1B[1mCreating the Subscription...\n${DB2_SUBSCRIPTION_FILE}\n\x1B[0m"
-kubectl apply -f ${DB2_OPERATOR_GROUP_FILE}
 
-echo "Sleeping for 5 minutes"
-sleep 150
 
+###### Create subscription to Db2 Operator
+#echo -e "\x1B[1mCreating the Subscription...\n${DB2_SUBSCRIPTION_FILE}\n\x1B[0m"
+#kubectl apply -f ${DB2_OPERATOR_GROUP_FILE}
 
+wating_time=20
 echo
-echo "Waiting up to 5 minutes for DB2 Operator install plan to be generated."
+echo "Waiting up to ${wating_time} minutes for DB2 Operator install plan to be generated. $DB2_PROJECT_NAME"
```

Comment:
`wating_time` is misspelled.  Double check where this is used.

Reply:
echo "Waiting up to ${waiting_time} minutes for DB2 Operator install plan to be generated...